single_kernel_mongo/config/literals.py

```diff
@@ -98,3 +98,7 @@ class VmUser(WorkloadUser[int]):
 MAX_PASSWORD_LENGTH = 4096
 
 PBM_RESTART_DELAY = 5
+
+OS_REQUIREMENTS = {
+    "vm.max_map_count": "262144",
```

Comment:
maybe a few more sysctls have sense to add, but needs research - https://www.percona.com/blog/tuning-linux-for-mongodb/

Reply:
-> https://warthogs.atlassian.net/browse/DPE-6419